system/i18n/formbuilder/item-types/date.properties

```diff
@@ -25,4 +25,7 @@ fieldset.relative.title=Relative dates
 relativeOperator.lt=Less than...
 relativeOperator.lte=Less than or equal to...
 relativeOperator.gt=Greater than...
-relativeOperator.gte=Greater than or equal to...
\ No newline at end of file
+relativeOperator.gte=Greater than or equal to...
+
+field.offset.title=Offset(Days)
```

Comment:
Should have space between `Offset` and `(Days)`. So `Offset (days)`. Help should be:

> For relative dates, supply an optional offset to allow date restrictions such as 14 days greater than today.